client/components/button/style.scss

```diff
@@ -63,7 +63,15 @@ button {
 			border-color: $blue-medium;
 			box-shadow: 0 0 0 2px $blue-light;
 	}
+
+	&.is-desktop,
 	&.is-compact {
+		.gridicon + span {
+			margin-left: 4px;
+		}
+	}
+
+	&.is-compact  {
```

Comment:
Extra space in here before the opening bracket. Curious if you've [set up ESLint](https://wpcalypso.wordpress.com/devdocs/docs/coding-guidelines/javascript.md#eslint) in whichever editor you're using? Super helpful for small things like this.